tests/foreman/api/test_computeresource_azurerm.py

```diff
@@ -36,8 +34,13 @@ class TestAzureRMComputeResourceTestCase:
 
     @pytest.mark.upgrade
     @pytest.mark.tier1
+    @pytest.mark.parametrize('sat_azure', ['sat', 'puppet_sat'], indirect=True)
```

Comment:
It would be nice if include ids here
```suggestion
    @pytest.mark.parametrize('sat_azure', ['sat', 'puppet_sat'], indirect=True, ids=['satellite', 'puppet_enabled'])
```

Reply:
yup